backend-nannyway/src/test/java/de/neuefische/nannyway/controller/ChildInCareControllerTest.java

```diff
@@ -0,0 +1,91 @@
+package de.neuefische.nannyway.controller;
+
+import de.neuefische.nannyway.database.ChildInCareMongoDb;
+import de.neuefische.nannyway.database.UserDb;
+import de.neuefische.nannyway.model.AddChildInCareDto;
+import de.neuefische.nannyway.model.ChildInCare;
+import de.neuefische.nannyway.model.LoginData;
+import de.neuefische.nannyway.model.NannywayUser;
+import de.neuefische.nannyway.security.JwtUtils;
+import de.neuefische.nannyway.utils.RandomIdUtils;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.http.*;
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+import java.time.LocalDate;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.when;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+class ChildInCareControllerTest {
```

Comment:
i would also test that you get forbidden when you are not logged in

Reply:
done